src/app/GitUI/UserControls/RevisionGrid/Columns/BuildStatusColumnProvider.cs

```diff
@@ -19,7 +19,7 @@ internal sealed class BuildStatusColumnProvider : ColumnProvider
         private readonly RevisionDataGridView _gridView;
         private readonly Func<IGitModule> _module;
 
-        // Increase contrast to selected rows
+        // Increase contrast to selected rows (adopted to theme at use)
```

Comment:
"adopt" or "adapt"? 
"adopted to theme" does not sound correct to me. One can adopt something.

Reply:
will change all occurrences to adapt....